kahenya-anita/Bumble-Clone
# Footer anita kahenya

## Description
Changed HTML folder structure, moved from nested divs to using list element and pushed the styles of the footer component to the root CSS file.
## Ticket [Footer](https://sharing.clickup.com/36234712/t/h/8669fja8v/0KDZQUXOBCBU5GD)
## Types Of Changes
[] Bug fix

[] New Feature

[x] Breaking Changes

## Checklist:
[x]My code follows the code style of this project

[x]All new and existing tests passed.

[x]My code has been linted and formatted correctly.

[x]I have commented on my code, particularly in hard-to-understand areas.

[x]My changes generate no new warnings.

## How can this code be tested?
1. After Cloning this repo , run `git pull
` to ensure you are up to date.

![Screenshot 2023-03-09 at 3 11 50 PM](https://user-images.githubusercontent.com/62019551/224019614-cad22c4b-5660-4ac2-be1a-f58acda72cb6.png)

2. Checkout to (Footer_Anita-Kahenya)[Footer_Anita-Kahenya] branch

![Screenshot 2023-03-09 at 3 12 55 PM](https://user-images.githubusercontent.com/62019551/224019880-30ed71b8-642e-45c8-9c7a-340bd629d137.png)

4. Run the changes in the browser

![Screenshot 2023-03-09 at 3 16 45 PM](https://user-images.githubusercontent.com/62019551/224020628-de671075-01a3-46a6-bca6-c3cc8ee68cd8.png)
